modules/rosa-nodes-machine-pools-local-zones.adoc

```diff
@@ -0,0 +1,73 @@
+
+// Module included in the following assemblies:
+//
+// * assemblies/rosa-nodes-machinepools-configuring.adoc
+:_content-type: PROCEDURE
+[id="rosa-nodes-machine-pools-local-zones_{context}"]
+= Configuring Local Zones for machine pools
+
+Use the following steps to configure Local Zones for machine pools.
+
+[IMPORTANT]
+====
+AWS Local Zones are supported on Red Hat OpenShift Service on AWS 4.12. See the link:https://access.redhat.com/articles/6989889[Red Hat Knowledgebase article] for information on how to enable Local Zones. 
+====
+.Prerequisites
+
+* ROSA is generally available in the parent region of choice. See the link:https://aws.amazon.com/about-aws/global-infrastructure/localzones/locations/?pg=ln&cp=bn#GA[AWS generally available locations list] to determine the Local Zone available to specific AWS regions.
+* The ROSA cluster was initially built in an existing Amazon VPC (BYO-VPC).
+* The maximum transmission unit (MTU) for the ROSA cluster is set at 1200.
++
+[IMPORTANT]
+====
+Generally, the Maximum Transmission Unit (MTU) between an Amazon EC2 instance in a Local Zone and an Amazon EC2 instance in the Region is 1300. See link:https://docs.aws.amazon.com/local-zones/latest/ug/how-local-zones-work.html[How Local Zones work] in the AWS documentation. 
+The cluster network MTU must always be less than the EC2 MTU to account for the overhead. The specific overhead is determined by your network plugin, for example:
+
+- OVN-Kubernetes: `100 bytes`
+- OpenShift SDN: `50 bytes`
+
+The network plugin could provide additional features that may also decrease the MTU. Check the documentation for additional information.
+
+====
+* The AWS account has link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-find-local-zone[Local Zones enabled].
+* The AWS account has a link:https://docs.aws.amazon.com/local-zones/latest/ug/getting-started.html#getting-started-create-local-zone-subnet[Local Zone subnet] for the same VPC as the cluster.
+* The AWS account has a subnet that is associated with a routing table that has a route to a NAT gateway.
+* The AWS account has the tag `kubernetes.io/cluster/<infra_id>: shared' on the associated subnet.
+
+.Procedure
+
+. Create a machine pool on the cluster by running the following ROSA CLI command.
++
+[source,terminal]
+----
+$ rosa create machinepool -c <cluster-name> -i 
+----
++
+. Add the subnet and instance type for the machine pool in ROSA CLI. After several minutes, the cluster will provision the nodes.
++
+[source, terminal]
+----
+I: Enabling interactive mode <1>
+? Machine pool name: xx-lz-xx <2>
+? Create multi-AZ machine pool: No <3>
+? Select subnet for a single AZ machine pool (optional): Yes <4>
+? Subnet ID: subnet-<a> (region-info) <5>
+? Enable autoscaling (optional): No <6>
+? Replicas: 2 <7>
+I: Fetching instance types <8>
+----
++
+
```

Comment:
```suggestion
```
Verify that this doesn't work better for you. Callouts typically follow code blocks immediately.

Reply:
Removing this changes the numbering structure, so it needs to remain.